src/query/query_executor.c

```diff
@@ -2248,6 +2249,11 @@ qexec_clear_xasl (THREAD_ENTRY * thread_p, xasl_node * xasl, bool is_final)
       pg_cnt += qexec_clear_agg_orderby_const_list (thread_p, xasl, is_final);
     }
 
+  if (xasl->sq_cache != NULL)
+    {
+      sq_cache_destroy (xasl);
```

Comment:
The xasl stored in xcache_entry->cache_clones is reused.
When generating xasl, information set in sq_cache must not be cleared.

Reply:
I've handled that error. Please confirm.